plothist/hep_plotters.py

```diff
@@ -160,6 +160,23 @@ def compare_data_mc(
                     np.sqrt(_hist_ratio_variances(data_hist_high, mc_hist_total)),
                 ],
             )
+    elif comparison_kwargs["comparison"] == "difference":
+        data_hist_high = data_hist.copy()
+        data_hist_high[:] = np.stack(
+            [data_hist_high.values(), uncertainties_high ** 2], axis=-1
+        )
+        data_hist_low = data_hist.copy()
+        data_hist_low[:] = np.stack(
+            [data_hist_low.values(), uncertainties_low ** 2], axis=-1
+        )
```

Comment:
The 8 lines above are copied and pasted from just above, we should try to avoid this.

Reply:
I think it's fine for now, but we can think of a optimized way for future version.